mender/templates/db-migration-job.yaml

```
@@ -202,6 +202,9 @@ spec:
 {{- if .Values.workflows.containerSecurityContext.enabled }}
         securityContext: {{- omit .Values.workflows.containerSecurityContext "enabled" | toYaml | nindent 10 }}
 {{- end }}
+        env:
+        - name: WORKFLOWS_NATS_URI
+          value: {{ template "nats_uri" . }}
```

Use of the built-in `template` function is discouraged since it cannot be used in a pipeline.
```suggestion
          value: {{ include "nats_uri" . }}
```